assets/css/main.css

```diff
@@ -2,623 +2,650 @@ html, body {
+  background:  rgba(204, 40, 41, 1);
+  border-radius:25px;
+}
+.btn-hire-us:hover{
+  cursor:pointer;
+  opacity:.95
+}
+
+a.btn-more{
+  display:block;
+  position:absolute;
+  bottom:25px;
+  left:50%;
+  margin-left:-25px;
+  width:50px;
+  height:50px;
+  cursor:pointer;
+  -webkit-animation: bounce 4s infinite;
+  animation: bounce 4s infinite;
 }
 
-/* MENU BUTTON */
-
-.menu-toggle {
-  display: none;
-  margin-top: 15px;
-  margin-right: 18px;
-  position: relative;
-  width: 51px;
-  height: 51px;
-  background: transparent;
-  cursor: pointer;
-}
-.menu-toggle .bar {
-  position: absolute;
-  width: 35px;
-  height: 5px;
-  background: #DFDDD7;
-  -webkit-transition: all 300ms ease;
-          transition: all 300ms ease;
-}
-.menu-toggle.after-fold .bar {
-  background: #282928;
-}
-.menu-toggle .bar:nth-child(1) {
-  left: 5px;
-  top: 4px;
-}
-.menu-toggle .bar:nth-child(2) {
-  left: 5px;
-  top: 16px;
-}
-.menu-toggle .bar:nth-child(2):after {
-  content: "";
-  position: absolute;
-  left: 0px;
-  top: 0px;
-  width: 35px;
-  height: 5px;
-  background: #DFDDD7;
-  -webkit-transition: all 300ms ease;
-          transition: all 300ms ease;
-}
-.menu-toggle.after-fold .bar:nth-child(2):after {
-  background: #282928;
-}
-.menu-toggle .bar:nth-child(3) {
-  right: 11px;
-  top: 28px;
-}
-.menu-toggle.open .bar:nth-child(1),
-.menu-toggle.open .bar:nth-child(3) {
-  width: 0;
-}
-.menu-toggle.open .bar:nth-child(2) {
-  -webkit-transform: rotate(-45deg);
-      -ms-transform: rotate(-45deg);
-          transform: rotate(-45deg);
-}
-.menu-toggle.open .bar:nth-child(2):after {
-  -webkit-transform: rotate(-90deg);
-      -ms-transform: rotate(-90deg);
-          transform: rotate(-90deg);
-}
-
-/* END MENU BUTTON */
-
-@media all and (max-width: 770px) {
-  .top-bar h1 {
-    font-size: 1.2rem;
-    margin: 15px 0 15px 20px;
-  }
-  .menu-toggle {
-    display: inline-block;
-  }
-  .header-nav-wrapper {
-    -webkit-flex-basis: 100%;
-        -ms-flex-preferred-size: 100%;
-            flex-basis: 100%;
-    height: auto;
-    display: none;
-  }
-  .header-nav-list {
-    -webkit-flex-basis: 100%;
-        -ms-flex-preferred-size: 100%;
-            flex-basis: 100%;
-    height: auto;
-    padding: 0;
-    -webkit-box-orient: vertical;
-    -webkit-box-direction: normal;
-    -webkit-flex-direction: column;
-        -ms-flex-direction: column;
-            flex-direction: column;
-    margin: 0;
-  }
-  .header-nav-list-item {
-    width: 100%;
-    text-align: center;
-    background: #DFDDD7;
-  }
-  .header-nav-list-item-link {
-    display: inline-block;
-    padding: 15px 0;
-    margin-left: 0;
-    width: 100%;
-    color: #554F4E;
-  }
-  .header-nav-list-item-link:hover {
-    color: #282928;
-  }
-  .header-nav-list-item-link.active {
-    border-bottom: none;
-  }
-  .header-nav-list-item-link.hire-us-link {
-    background-color: #F1853D;
-    color: #fff;
-    padding: 15px 0;
-  }
+.big-splash div a.btn-more:hover{
```

Comment:
could these just be `.big-splash .btn-more:hover`? and the one below `.big-splash .btn-more-inner` or something